react/features/filmstrip/actions.web.js

```diff
@@ -80,21 +85,64 @@ export function setVerticalViewDimensions() {
     return (dispatch: Dispatch<any>, getState: Function) => {
         const state = getState();
         const { clientHeight = 0, clientWidth = 0 } = state['features/base/responsive-ui'];
+        const { width: filmstripWidth } = state['features/filmstrip'];
         const disableSelfView = shouldHideSelfView(state);
-        const thumbnails = calculateThumbnailSizeForVerticalView(clientWidth);
+        const resizableFilmstrip = isFilmstripResizable(state);
+        const _verticalViewGrid = resizableFilmstrip && ((filmstripWidth.current ?? 0) > FILMSTRIP_GRID_BREAKPOINT);
+
+        let gridView = {};
+        let thumbnails = {};
+        let filmstripDimensions = {};
+
+        if (_verticalViewGrid) {
```

Comment:
Can you maybe add a one line comment about what specific layout this is "vertical view grid" reads kinda weird to me.

Reply:
Sure. I see your point, but vertical filmstrip is named vertical view in the layouts so I tried to keep it consistent